### PR TITLE
CKAN 2.10

### PR DESCRIPTION
**Pull request by Jin-Sun-tts**
Related to
- https://github.com/GSA/data.gov/issues/4209

Changes:
- update test because 2.10 randomizes organization name.
- set default testing to 2.10
- change datagovtheme to ckan-210 temporarily.

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
Tests are passing.. TODO:
- [x] Update README
- [x] Sync `CKAN 2.10` templates
- [x] Any other random changes... TBD

**Review by FuhuXia — DISMISSED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -e git+https://github.com/ckan/ckanext-report.git@master#egg=ckanext-report
 -e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest
 
-ckanext-datagovtheme
+-e git+https://github.com/gsa/ckanext-datagovtheme.git@ckan-210#egg=ckanext-datagovtheme
```
Comment (jbrown-xentity) [inline requirements.txt]: We should probably merge https://github.com/GSA/ckanext-datagovtheme/pull/162/files, and then leave this line as it was...

**Review by jbrown-xentity — COMMENTED** [review]
Mostly good, just a small change!

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ARG CKAN_VERSION=2.9
+ARG CKAN_VERSION=2.10
```
Comment (btylerburton) [inline Dockerfile:1]: should value be `2.10.1`?

File: Makefile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-CKAN_VERSION ?= 2.9
+CKAN_VERSION ?= 2.10
```
Comment (btylerburton) [inline Makefile:1]: should value be `2.10.1`?

File: docker-compose.yml [hunk 4]
```diff
@@ -31,7 +31,7 @@ services:
       - ./seed.py:/srv/app/seed.py
       - /etc/timezone:/etc/timezone:ro
   db:
-    image: ckan/ckan-postgres-dev:${CKAN_VERSION}
+    image: ckan/ckan-postgres-dev:2.9
```
Comment (btylerburton) [inline docker-compose.yml]: do we need bump to `2.10`?

File: requirements.txt (hunk 1 above)
Comment (btylerburton) [reply requirements.txt]: +1

**Review by btylerburton — COMMENTED** [review]
a few comments...

**Review by nickumia-reisys — DISMISSED** [review]
🧪

File: Dockerfile (hunk 2 above)
Comment (FuhuXia) [reply Dockerfile:1]: no need to pin to 2.10.1

File: Makefile (hunk 3 above)
Comment (FuhuXia) [reply Makefile:1]: 2.10 is ok, no need to pin to 2.10.1

**Review by nickumia-reisys — APPROVED** [review]
🧪 🧪

**Pull request merged**